template-manager-cmtemplate/src/main/java/com/sequenceiq/cloudbreak/cmtemplate/configproviders/hive/HiveMetastoreConfigProvider.java

```diff
@@ -87,6 +90,12 @@
         );
         addDbSslConfigsIfNeeded(templateProcessor, hiveRdsView, configs, getCmVersion(source));
 
+        // For DataHub, don't start the compactor.Initiator thread which automatically queues ACID compaction.
+        // The Initiator thread in the DataLake's HMS will take care of this.
+        if (source.getStackType() == StackType.WORKLOAD) {
+            configs.add(config(HIVE_COMPACTOR_INITIATOR_ON, Boolean.FALSE.toString()));
```

Comment:
Is this config possibly tied to some specific CR version, or rather expected to work for everything from 7.x onwards?

Reply:
Great question! I believe this should have been changed starting at 7.0.0, but we just got around to doing it.

Reply:
Okay, then let us assume it will just work without a CR version check. 😃

Reply:
@klcopp can you please look up the commit when this has changed? We're not really testing old runtimes and the error would only come up in prod for customers if they want to launch new data hubs with an older version. 7.1.0 is our oldest in prod today.

Reply:
`the commit when this has changed?`
Do you mean the Hive commit that introduced the configuration? It's been there since before CDP 7.
Or do you mean when it was turned on in CM?

Reply:
The CM part.

Reply:
Ah ok. It's set n HMS starting at CDH 7.1.1. Does this mean that this DataHub CM change should apply only from CDH 7.1.1?

Reply:
Yes, in this case you need to introduce a version check for this property. You can find examples in the other role config provider on how to do that.

Reply:
One example: https://github.com/hortonworks/cloudbreak/blob/master/template-manager-cmtemplate/src/main/java/com/sequenceiq/cloudbreak/cmtemplate/configproviders/raz/RangerRazDatalakeConfigProvider.java#L27
Feel free to introduce a new helper method in `CMRepositoryVersionUtil` that checks for CM version 7.1.1.

Another example that uses a different approach: https://github.com/hortonworks/cloudbreak/blob/master/template-manager-cmtemplate/src/main/java/com/sequenceiq/cloudbreak/cmtemplate/configproviders/hive/HiveMetastoreConfigProvider.java#L146